.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build generator-symphony NodeJs project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs-or-python
+
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 17.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm ci
```

Comment:
We should use yarn IMO

Reply:
Done in https://github.com/finos/generator-symphony/pull/169/commits/609037006fd50a1533c520960a3d37d58bd07301